<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="robot_name"/>
    <!-- <arg name="obstacle_name1"/> -->
    <arg name="rviz" default="true"/>
    <rosparam command="load" file="$(find fabrics_bridge)/config/fabrics_kuka_config.yaml"/>
<!-- 
    <group ns="fabrics"> -->
        <!-- start fabric node -->
    <node pkg="fabrics_bridge" type="kuka_fabrics_node" name="kuka_fabrics_node" output="screen"/>
    <!-- </group> -->
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find fabrics_bridge)/config/fabrics_boxer_visualization.rviz" if="$(arg rviz)"/> -->
    <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find dingo_planners)/rviz/demo.rviz"/> -->
    <!-- Basic visualization
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find fabrics_bridge)/rviz/demo.rviz"/> -->
</launch>
